app/views/eventDetails/paymentOptions.html

```diff
@@ -5,6 +5,15 @@ <h2 class="page-title border" translate>Payment Options</h2>
     <h3 class="panel-title" translate>Credit Card</h3>
   </div>
   <div class="panel-body">
+    <div class="form-group">
+      <div class="col-sm-12 stacked-spacing-col-sm" ng-if="conference.paymentGatewayType === 'AUTHORIZE_NET'">
+        <label translate>Select A Payment Gateway Type:</label>
+        <select class="form-control" ng-model="conference.paymentGatewayType">
+          <option value="TSYS" translate>TSYS Merchant ID</option>
```

Comment:
I was going strictly off of what IM-203 Jira says, "two options, 'Authorize.NET' and 'TSYS Merchant ID'."
So I used 'Authorize.NET' and 'TSYS Merchange ID'.
Do you think it would be better for end users to make this select option 'TSYS'?